native/lambda_game_engine/src/player.rs

```diff
@@ -272,6 +276,24 @@ impl Player {
         }
         None
     }
+
+    pub fn put_in_inventory(&mut self, loot: &Loot) -> bool {
+        match self
+            .inventory
+            .iter_mut()
+            .find_position(|element| element.is_none())
+        {
+            Some((slot_at, _)) => {
+                self.inventory[slot_at] = Some(loot.clone());
+                true
+            }
+            None => false,
+        }
+    }
+
+    pub fn inventory_get_at(&mut self, inventory_at: usize) -> Option<Loot> {
+        self.inventory[inventory_at].take()
+    }
```

Comment:
I would expect a `X_get_at` function to get the value and do nothing with self. Maybe `inventory_take_at` is less confusing, what do you think?

Reply:
That makes a lot of sense, renamed in e2ca46f